### ISSUE TITLE
Ability to sync with Stellar as well

### ISSUE THREAD

**Issue by mysticuno** [popoffka/lmoddiff#1]
I'm not sure if this is possible with Stellar, but it'd be nice to have some of these compatible with Stellar as well since not all classes have migrated to LMOD yet.

**Comment by aleksejspopovs** [comment]
I actually haven't ever seen any classes use Stellar for grades. Is that even possible?

All of the classes I'm taking this term are LMOD-only. Last term, 5.111 posted the assignments themselves on Stellar, but put the grades on LMOD, which was kind of annoying and would probably be a huge pain to implement (especially since there was no consistency in how the psets were named on Stellar, so, IIRC, I would have “Problem Set 4” followed by “Problem set 5” followed by “Problem set #6” etc). In that situation, lmoddiff in its current iteration would still be somewhat useful because it'd still notify you that an assignment has been created on LMOD, it would just incorrectly tell you that the PDF for the assignment isn't there.

So yeah, I am currently unable to implement this, and am not sure if it's even possible/a good idea, but, if anyone has more information/wants to contribute code, I'm open to that.

**Comment by mysticuno** [comment]
Yup, there are still some classes that do this. 24.900 is one example of
it. Last semester 6.021 also had its grades on stellar. However, I'm not
sure if Stellar has a nice API like LMOD does, and it seems to me that MIT
is slowly trying to move everything onto LMOD, so going forward I agree
that a feature like this might become obsolete and perhaps not worth the
time implementing right now.
